sherpa/conftest.py

```diff
@@ -42,13 +42,29 @@
     has_xspec = False
 
 
-import sherpa.plot
-import sherpa.astro.plot
-from sherpa.plot import dummy_backend
+from sherpa.plot.backends import BaseBackend, IndepOnlyBackend, PLOT_BACKENDS
```

Comment:
```suggestion
from sherpa.plot.backends import PLOT_BACKENDS
```

I don't think we need to import these two symbols, unless there's something magical happening, and given that this is a `pytest` file, there's always something magical going on.

Reply:
We do need to import them, because they are added to the registry when they are imported first.